Source/Windows/ARM64EC/CMakeLists.txt

```diff
@@ -23,7 +23,7 @@ target_link_libraries(arm64ecfex
   ntdll_ex
 )
 
-target_link_options(arm64ecfex PRIVATE -static -nostdlib -nostartfiles -nodefaultlibs -lc++ -lc++abi -lunwind -lclang_rt.builtins-arm64ec)
+target_link_options(arm64ecfex PRIVATE -static -nostdlib -nostartfiles -nodefaultlibs -lc++ -lc++abi -lunwind ${libgcc_path})
```

Comment:
CMake variables should be ALL_UPPERCASE. We don't use lowercase variables anywhere else.

Reply:
Fixed in the new version.